src/Misc/layoutbin/en-US/strings.json

```diff
@@ -163,6 +163,7 @@
   "Password": "password",
   "PathNotExist": "Path not exist: {0}",
   "PersonalAccessToken": "personal access token",
+  "PlatformNotSupport": "Current platform is not capable of running agent that build for {0}.",
```

Comment:
Running the agent on this platform is not supported.  The current platform is {0} and it was built for {1}